ds-caselaw-ingester/lambda_function.py

```diff
@@ -447,15 +448,13 @@ def set_document_identifiers(self) -> None:
             msg = f"Ingesting, but identifiers already present for {self.uri}!"
             logger.warning(msg)
 
-        try:
-            ncn = doc.neutral_citation
-        except AttributeError:
-            ncn = None
+        ncn = doc.neutral_citation
+        identifier_class = PressSummaryRelatedNCNIdentifier if "/press-summary" in self.uri else NeutralCitationNumber
```

Comment:
The presence of anything in the URI isn't safe to rely on (in theory the parser should be able to send us completely unstructured URIs); we need to be checking the `type` in the document XML directly.